src/main/java/it/pagopa/ecommerce/commons/client/NpgClient.java

```diff
@@ -644,8 +710,34 @@ private CreateHostedOrderRequestDto buildOrderRequestDto(
                                                              String customerId,
                                                              PaymentMethod paymentMethod,
                                                              String contractId,
-                                                             Integer totalAmount
+                                                             Integer totalAmount,
+                                                             String language
     ) {
+
+        String ISO_639_3_lang = CREATE_HOSTED_ORDER_REQUEST_LANGUAGE_ITA;
+        if (language != null) {
+            switch (language.toLowerCase()) {
+                case "it":
+                    ISO_639_3_lang = "ita";
+                    break;
+                case "fr":
+                    ISO_639_3_lang = "fra";
+                    break;
+                case "en":
+                    ISO_639_3_lang = "eng";
+                    break;
+                case "de":
+                    ISO_639_3_lang = "deu";
+                    break;
+                case "sl":
+                    ISO_639_3_lang = "slv";
+                    break;
+                default:
+                    ISO_639_3_lang = "ita";
+                    break;
+            }
+        }
```

Comment:
We could use a static `Map` instead, what do you think?

```java
private static final LANG_MAP = Map.of(
    "it", "ita",
    "fr", "fra",
    "de", "deu"
);
```

and then

```suggestion
        if (language != null) {
            ISO_639_3_lang = LANG_MAP.getOrDefault(language, CREATE_HOSTED_ORDER_REQUEST_LANGUAGE_ITA);
        }
```